### DISCUSSION TITLE
Backwards Mac compatibility ( 10.13)

### DISCUSSION THREAD

**Discussion by kempernorton** [helgoboss/helgobox#1057] [Ideas]
Us vintage Mac users want to use Playtime 2 too!

**Comment by helgoboss** [comment]
Does the Helgobox plug-in work on your machine (= basically ReaLearn)?

**Comment by kempernorton** [comment]
It appears as a plugin and gives me a screen to create inputs, outputs and
mapping etc

**Reply by helgoboss** [reply]
Ok thanks. So it's just a matter of getting the app (the pretty GUI) to work. I hope to be able to do it at some point.

**Comment by kempernorton** [comment]
Thanks Benjamin
